### ISSUE TITLE
Dropwizard support for 1.0.0

### ISSUE THREAD

**Issue by jmachalica** [prometheus/client_java#867]
Is it possible to integrate this client version with dropwizard? I would like to expose its metrics for prometheus scraping. Should I use an archived simple client?

**Comment by fstab** [comment]
Hi @jmachalica, the `simpleclient_dropwizard` module didn't make it into the 1.0.0 release, it's still on version 0.16.0. We are planning to port it over, but it will take some time. You have three options:

  1. Keep using 0.16.0 until a 1.x version of `simpleclient_dropwizard` is available. There's nothing wrong with 0.16.0.
  2. If you want to use the new 1.0.0 version for new metrics but keep `simpleclient_dropwizard` for Dropwizard metrics, you can use both versions in parallel. Use the `prometheus-metrics-simpleclient-bridge` module as described here: https://prometheus.github.io/client_java/migration/simpleclient/.
  3. We are looking for volunteers to help us port the remaining 0.16.0 modules over to 1.0.0. So if you want to speed things up, feel free to create a PR. All you need is a `MultiCollector` for Dropwizard metrics that produces snapshots as defined in `prometheus-metrics-model`, see https://prometheus.github.io/client_java/internals/model/. I'm happy to help.

**Comment by bmeier-pros** [comment]
Can we split this into metrics 4 and metrics 5 tickets?  The change merged only supports metrics 5, we need metrics 4 for Dropwizard 3 and 4 support.

**Comment by twbecker** [comment]
Agree Dropwizard users have kind of been left in the lurch with the 1.0 release. Dropwizard 5 isn't released and doesn't even seem imminent.

**Comment by kingster** [comment]
Let me take a stab at dw metrics4 and see what I get working with. 

One possible option is to duplicate the code but that is what I don't like, so need to figure out how the exporter can work with both versions.

Any suggestions? @fstab

**Comment by kingster** [comment]
@twbecker @bmeier-pros 

Dropwizard metrics 4.x version is now supported via the `io.prometheus:prometheus-metrics-instrumentation-dropwizard` package.

**Comment by bmeier-pros** [comment]
@kingster - one comment, it appears the new module is missing from the bom for 1.3.5.

**Comment by zeitlinger** [comment]
> @kingster - one comment, it appears the new module is missing from the bom for 1.3.5.

see https://github.com/prometheus/client_java/issues/1249
